src/lib/obfuscator/transforms/transforms/bogus_control_flow.hpp

```diff
@@ -52,9 +52,9 @@ namespace obfuscator::transforms {
                 /// Generating a BCF stub
                 transform_util::generate_bogus_confrol_flow<Img>(
                     function, bb.get(),
-                    [&](std::shared_ptr<analysis::bb_t> new_bb) -> void {
+                    [&](const std::shared_ptr<analysis::bb_t>& new_bb) -> void {
```

Comment:
dk if we need std::shared_ptr reference here, mb it's better to just pass `analysis::bb_t*`?